module/localisation/BallLocalisation/src/BallModel.cpp

```diff
@@ -66,11 +66,12 @@ namespace localisation {
     }
 
     arma::vec::fixed<BallModel::size> BallModel::limitState(const arma::vec::fixed<size>& state) const {
-        return state;
+        // TODO: configure
```

Comment:
This looks like something that will come back to bite us if we don't deal with it now. Nobody is going to remember this thing in a few weeks/months time.